showcase/src/app/demo-rbac/pages/authorization-directives/demo-authorization-directives-page.component.ts

```diff
@@ -0,0 +1,61 @@
+import { Component, Inject } from "@angular/core";
+import { MatButtonToggleChange } from "@angular/material/button-toggle";
+import { StarkUser, STARK_SESSION_SERVICE, StarkSessionService } from "@nationalbankbelgium/stark-core";
+import { StarkRBACDirectivePermission } from "@nationalbankbelgium/stark-rbac";
+import { filter, take } from "rxjs/operators";
+import { ReferenceLink } from "../../../shared/components/reference-block";
+
+@Component({
+	selector: "demo-authorization-directives",
+	templateUrl: "./demo-authorization-directives-page.component.html",
+	styleUrls: ["./demo-authorization-directives-page.component.scss"]
+})
+export class DemoAuthorizationDirectivesPageComponent {
+	public selectedAuthorizedRole: string;
+	public selectedUnauthorizedRole: string;
+
+	public referenceList: ReferenceLink[] = [
+		{
+			label: "Stark Show On Permission directive",
+			url: "https://stark.nbb.be/api-docs/stark-rbac/latest/directives/StarkShowOnPermissionDirective.html"
+		},
+		{
+			label: "Stark Hide On Permission directive",
+			url: "https://stark.nbb.be/api-docs/stark-rbac/latest/directives/StarkHideOnPermissionDirective.html"
+		}
+	];
+
+	public get onHidePermissionsConfig(): StarkRBACDirectivePermission {
+		return {
+			roles: [this.selectedUnauthorizedRole]
+		};
+	}
+
+	public get onShowPermissionsConfig(): StarkRBACDirectivePermission {
+		return {
+			roles: [this.selectedAuthorizedRole]
+		};
+	}
+
+	public constructor(@Inject(STARK_SESSION_SERVICE) sessionService: StarkSessionService) {
+		// taking the role of the current user to initialize the toggle button
+		sessionService
+			.getCurrentUser()
+			.pipe(
+				filter<StarkUser | undefined, StarkUser>((user?: StarkUser): user is StarkUser => typeof user !== "undefined"),
+				take(1)
+			)
+			.subscribe((user: StarkUser) => {
+				this.selectedAuthorizedRole = user.roles[0];
```

Comment:
Since we define the mat-button-toggle in the HTML with **manager** (which is the role of the default user),
```html
<mat-button-toggle value="manager">Manager</mat-button-toggle>
```
I think we should remove this code because it's useless.

We set the role **manager** in the route configuration too. So, in any case, this part cannot be really dynamic.

Reply:
Indeed, I'll remove it 👍